lib/scry/sidekiq/boot.rb

```diff
@@ -0,0 +1 @@
+Dir["./lib/scry/sidekiq/workers/*.rb"].each { |file| require file }
```

Comment:
Consider adding a file in `bin/` that calls this file? So when we can `bundle exec sidekiq -r ./bin/boot.rb -C sidekiq.yml` Or even better, a script in `bin/` that does that for you? `bin/boot_sidekiq` ?